VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.ts

```diff
@@ -322,4 +316,16 @@ export class ParticipantWaitingRoomComponent implements OnInit {
   toggleView(): boolean {
     return this.selfViewOpen = !this.selfViewOpen;
   }
+
+  getConferenceClosedTime(conferenceId: string): void {
+    this.videoWebService.getConferenceById(conferenceId)
+      .subscribe(async (data: ConferenceResponse) => {
+        this.hearing = new Hearing(data);
+        this.conference = this.hearing.getConference();
+        this.logger.info(`Participant waiting room for conference: ${conferenceId} and participant: ${this.participant.id}`);
```

Comment:
also update the participant here like 'getConference'

Reply:
added the participant